internal/configuration/validator/const.go

```diff
@@ -25,6 +25,8 @@ const (
 		"must be one of: '%s'"
 	errFmtOIDCServerClientInvalidResponseMode = "OIDC Client with ID '%s' has an invalid response mode '%s', " +
 		"must be one of: '%s'"
+	errFmtOIDCServerInsecureParameterEntropy = "SECURITY ISSUE: OIDC minimum parameter entropy is configured to an " +
+		"unsafe value, it should be above 8 but it's configured to %d."
```

Comment:
```suggestion
		"unsafe value, it should be above 8 but it's configured to %d."
```
If I'm wrong about the first one then this should be adjusted too, otherwise this is good.

Reply:
You're not, it should be above 8, nice catch.